### DISCUSSION TITLE
Need asio build

### DISCUSSION THREAD

**Discussion by notbadplayer** [GrandOrgue/grandorgue#1676] [Help]
hi, could someone build me the latest version with ASIO?  I tried to do it myself but despite several attempts I failed.  The available guide is no longer up to date (repositories are out of date).  Please, if any of you could build an ASIO version for me and share it, for example, on Google Drive or other hosting, I would be very grateful.  Regards

**Comment by larspalo** [comment]
Can't you just use a build from the intermediate action runs? https://github.com/GrandOrgue/grandorgue/actions or from the currently latest windows build at https://github.com/GrandOrgue/grandorgue/actions/runs/6395770746 which should include ASIO support...

**Comment by oleg68** [comment]
> hi, could someone build me the latest version with ASIO? I tried to do it myself but despite several attempts I failed. The available guide is no longer up to date (repositories are out of date). Please, if any of you could build an ASIO version for me and share it, for example, on Google Drive or other hosting, I would be very grateful. Regards

All windows builds at https://github.com/GrandOrgue/grandorgue/releases are ASIO-enabled.
